packages/server-bridge/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@stellarjs/bridge",
+  "name": "@stellarjs/server-bridge",
```

Comment:
I'm not sure about this name.
Maybe it can be `proxy` (if it just pass the messages) or `gateway` (if it does any validations).
Also, the name should hint that the this package is a `pipe between two transports`

Reply:
@roeeyud @ozsay what say you?